pb_app/lib/workflows.dart

```diff
@@ -2,16 +2,17 @@ import 'package:flutter/widgets.dart';
 import 'package:pb_app/config.dart';
 import 'package:pb_app/modals.dart';
 import 'package:pb_app/secrets.dart';
+import 'package:pb_app/submission_form.dart';
 import 'package:pocketbase/pocketbase.dart';
 
 final client = PocketBase(Config.baseUrl);
 
 class Workflows {
-  static Future<UserAuth> authenticate() {
-    return client.users.authViaEmail(
-      Secrets.email,
-      Secrets.password,
-    );
+  static Future<UserAuth> authenticate(email, password) {
+    if (email != null && password != null) {
+      return client.users.authViaEmail(email, password);
+    }
+    return client.users.authViaSecrets;
```

Comment:
We can remove Workflows.authenticate() method now that we have the authViaSecrets extension :)

Reply:
I think we need it for the login modal

Reply:
in login modal, can we do something like

```
if Config.skipLogin
  authViaSecrets
else
  authViaEmail

Reply:
we can, but I dont see whats wrong with this. 

Ill do it anyways though